client/src/utils/search.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+export const searchOnliner = (query)  => {
```

Comment:
the client has to send requests to back-end and should not be aware of what data source it uses under the hood.

Reply:
Yes, I meant that. That request will go to back-end, the route there is not ready yet.

Reply:
right, so FE should not contain such BE-related specifics in the code (onliner I mean)